src/pages/AccountPage.tsx

```diff
@@ -1,5 +1,276 @@
+            placeholder="Enter your email"
+            onChange={handleChange}
+          />
+        </div>
+        <Link to={'/'} cursor-pointer>
+          <Button text="Save Changes" />
+        </Link>
+      </section>
+
+      {/* Delivery Information */}
+      <section className="border-primary/20 mt-6 w-full max-w-2xl rounded-xl border bg-[#FAF6EC] p-6 shadow">
+        <h2 className="text-primary font-subtext mb-4">Delivery Information</h2>
+        <div className="grid w-full grid-cols-1 gap-4 md:grid-cols-2">
+          <InputWithLabel
+            id="firstName2"
+            label="FIRST NAME"
+            value={formData.firstName}
+            name="firstName"
+            type="text"
+            placeholder="Enter your first name"
+            onChange={handleChange}
+          />
+          <InputWithLabel
+            id="lastName2"
+            label="LAST NAME"
+            value={formData.lastName}
+            name="lastName"
+            type="text"
+            placeholder="Enter your last name"
+            onChange={handleChange}
+          />
+          <InputWithLabel
+            id="phone2"
+            label="PHONE NUMBER"
+            value={formData.phone}
+            name="phone"
+            type="tel"
+            placeholder="+1 (000) 000-0000"
+            onChange={handleChange}
+          />
+          <InputWithLabel
+            id="email2"
+            label="EMAIL"
+            value={formData.email}
+            name="email"
+            type="email"
+            placeholder="Enter your email"
+            onChange={handleChange}
+          />
+        </div>
+
+        <div className="m-auto w-full pb-4">
+          <InputWithLabel
+            id="address"
+            label="DELIVERY ADDRESS"
+            value={formData.address}
+            name="address"
+            type="text"
+            placeholder="Enter delivery address"
+            onChange={handleChange}
+          />
+        </div>
+
+        <h2 className="text-primary font-subtext mb-4">Additional Info</h2>
+
+        <div className="m-auto w-5/6 w-full pb-4">
+          <label htmlFor="notes" className="text-success">
+            ORDER NOTES (OPTIONAL)
+          </label>
+          <textarea
+            id="notes"
+            name="notes"
+            value={formData.notes}
+            onChange={handleChange}
+            placeholder="Notes about your order, e.g. special notes for delivery"
+            className="border-primary/30 w-full rounded-xl border-1 p-2"
+            rows={3}
+          />
+        </div>
+
+        <div className="mt-4 flex w-full flex-col gap-4 md:flex-row">
+          <Link to={'/'} cursor-pointer>
+            <Button text="Save Changes" />
+          </Link>
+          <button className="border-error text-error bg-background h-14 w-64 cursor-pointer rounded-full border px-4 py-2">
+            Delete
+          </button>
+        </div>
+      </section>
+
+      {/* aDDRESS 2 */}
```

Comment:
I would delete this whole section with the second address at least at this moment.